lib/libsys/Makefile.sys

```diff
@@ -100,22 +100,13 @@ INTERPOSED = \
 	write \
 	writev
 
-.ifdef CHERI_LIB_C18N
 # Pseudo implementations of system calls with special behavior under c18n
 SRCS+= \
 	mmap_c18n.c \
 	rfork_c18n.c \
 	thr_exit_c18n.c \
 	vfork_c18n.c
 
-PSEUDO+= \
```

Comment:
How does deleting this work?

Reply:
Not having the symbols in PSEUDO will cause them to be weak aliases of the default assembly stub. The *_c18n stubs can override them when defined.

Reply:
But those stubs need to be weak too, so how are you ensuring priority?

Reply:
Makes sense. The stubs currently are strong symbols but they certainly can be made weak.